diff_drive_controller/include/diff_drive_controller/odometry.h

```diff
@@ -142,9 +142,10 @@ namespace diff_drive_controller
     /**
      * \brief Sets the wheel parameters: radius and separation
      * \param wheel_separation Seperation between left and right wheels [m]
-     * \param wheel_radius     Wheel radius [m]
+     * \param wheel_radius     Left wheel radius [m]
```

Comment:
This is correct (updated) in my commit here: https://github.com/clearpathrobotics/ros_controllers/commit/9683ef19ec46e713ba42764355a0da2f2399300b

Reply:
It should be easy to cherry pick it.

Reply:
@artivis If you do this change you'll have exactly my commit in CPR. I could cherry pick it myself if you want and create another PR :)

Reply:
The change we are talking about is the doc diff you linked above ?

Reply:
AFAIK yes